### DISCUSSION TITLE
What is the best way to pose a model with AI without changing proportions?

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/Fooocus#3164] [General]
If we are talking about Fooocus, the two tools that has, PyraCanny and CPDS,  are quite good to copy a pose. But as they copy the shape, not only the pose, it will change the proportions of the character. Meaning if you are posing a cartoon character, you need a pose reference that matches those proportions or not only will pose it, also will change the head size etc. It will look similar but the proportions changed. Doing reference poses for all possible proportions would be too time consuming,
I think advanced AI Uis as ConfyUI etc have a specific way to do it using the pose only without involving proportions, based in the skeleton. But there is an easier UI to do this? 
The objective would be to use a specific pose reference and copy it to our model without changing proportions.

**Comment by ILikeToasters** [comment]
I use https://posemy.art/ often for when I need poses. I can then bring them over to Fooocus and use CPDS to generate my image.

**Reply by poor7** [reply]
Here is one way to use OpenPose from ControlNet in Fooocus
https://www.youtube.com/watch?v=HMvHDAYpcLU
